### PR TITLE
#2775 Adding popular collections to landing

### LINKED ISSUE #2775: Show popular trending collection on landing
Let's show people a recent trending collection, **sort the collection** **by most buys** in the **last 7 days** for easiness

Maybe this issue should be finished before but it's not blocking I guess  
- https://github.com/kodadot/nft-gallery/issues/2359

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2775
- [ ] Requires deployment <rubick/magick_issue>
- [x] We currently do not have timestamp and owner info in the query 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-06-03 at 6 18 33 PM

### CODE REVIEW

File: components/rmrk/Gallery/PopularCollections.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  protected async handleResult() {
+    const collections = await this.$apollo.query({
+      query: popularCollectionList,
+      client: 'subsquid',
+      variables: {
+        orderDirection: 'ASC',
+        limit: 10,
+        dateRange: '7 DAY',
```
Comment (kkukelka) [inline components/rmrk/Gallery/PopularCollections.vue:65]: should it be the most popular collections only of the past 7 days? just curious 😝

File: components/rmrk/Gallery/PopularCollections.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const {
+      data: { seriesInsightTable },
+    } = collections
+
+    this.nfts = seriesInsightTable.map(
+      (e: RowSeries): RowSeries => ({
+        ...e,
+        image: sanitizeIpfsUrl(e.image),
+      })
+    )
```
Comment (kkukelka) [inline components/rmrk/Gallery/PopularCollections.vue:79]: here it say the .map function will map seriesInsightTable to an array of RowSeries, yet above you defined `private nfts: any[] = []` . would be cool to set the correct type there

File: components/rmrk/Gallery/PopularCollections.vue (hunk 1 above)
Comment (kkukelka) [reply components/rmrk/Gallery/PopularCollections.vue:65]: by that i mean if it is correct you can just resolve this convo

**Review by kkukelka — CHANGES_REQUESTED** [review]
looks good. just 2 comments

File: components/rmrk/Gallery/PopularCollections.vue (hunk 1 above)
Comment (prachi00) [reply components/rmrk/Gallery/PopularCollections.vue:65]: yeah so it says in the issue that it has to be 7 days

File: components/rmrk/Gallery/PopularCollections.vue (hunk 2 above)
Comment (prachi00) [reply components/rmrk/Gallery/PopularCollections.vue:79]: Fixed this one @kkukelka

**Review by kkukelka — APPROVED** [review]
lgtm

**Comment by kkukelka** [comment]
pay 100

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 68.24 USD/KSM ~ 1.465 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x990260e498c20c2e3f547434246d738daa002bbe94c526f0c3c3ec6db85cfbe8](https://kusama.subscan.io/extrinsic/0x990260e498c20c2e3f547434246d738daa002bbe94c526f0c3c3ec6db85cfbe8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**